vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -271,7 +271,7 @@ static float NumberOrDefault(const winrt::Microsoft::ReactNative::JSValue &value
   return result;
 }
 
-static YGValue YGValueOrDefault(const winrt::Microsoft::ReactNative::JSValue &value, YGValue defaultValue) {
+static YGValue YGValueOrDefault(const winrt::Microsoft::ReactNative::JSValue &value, YGValue defaultValue, ShadowNodeBase& shadowNode, std::string key) {
```

Comment:
String parameters should generally not be passed by value. The story on what to use where in C++ is much more confusing than it should be, but for here, you could keep it as `const std::string&` instead of `std::string`.